README.md

```diff
@@ -54,7 +54,7 @@ crud.unflatten_rows(res.rows, res.metadata)
 
 * A space should have a format.
 * By default, `bucket_id` is computed as `vshard.router.bucket_id_strcrc32(key)`,
-  where `key` is the primary key value.
+  where `key` is the primary key value or a [sharding key set by DDL schema](https://github.com/tarantool/ddl#input-data-format).
```

Comment:
In operations below we define the `key` parameter as the primary key (for example, in `crud.get()` function description). It seems, those definitions should be updated as well to don't confuse users.

Reply:
Done.

```
--- a/README.md
+++ b/README.md
@@ -54,10 +54,12 @@ crud.unflatten_rows(res.rows, res.metadata)
 
 * A space should have a format.
 * By default, `bucket_id` is computed as `vshard.router.bucket_id_strcrc32(key)`,
-  where `key` is the primary key value or a [sharding key set by DDL schema](https://github.com/tarantool/ddl#input-data-format).
-  Custom bucket ID can be specified as `opts.bucket_id` for each operation.
-  For operations that accepts tuple/object bucket ID can be specified as
-  tuple/object field as well as `opts.bucket_id` value.
+  where `key` is a value of sharding key. Sharding key can be set with [DDL
+  schema](https://github.com/tarantool/ddl#input-data-format) (see
+  `sharding_key` option) or in a space `_ddl_sharding_key`. By default sharding
+  key is a primary key. Custom bucket ID can be specified as `opts.bucket_id`
+  for each operation. For operations that accepts tuple/object bucket ID can be
+  specified as tuple/object field as well as `opts.bucket_id` value.
 
 ### Insert
 
```

Reply:
I'll cite the README:

> ### Get
>
> ```lua
> local object, err = crud.get(space_name, key, opts)
> ```
>
> where:
>
> * `space_name` (`string`) - name of the space
> * `key` (`any`) - primary key value
> * `opts`:
> <...>

I'm interested in this line:

> * `key` (`any`) - primary key value

It is incorrect.